pkg/commands/compute/build.go

```diff
@@ -151,11 +169,29 @@ func (c *BuildCommand) Exec(in io.Reader, out io.Writer) (err error) {
 		}
 	}
 
-	if c.Manifest.File.Scripts.Build != "" {
-		lang = "custom"
+	// NOTE: A third-party could share a project with a build command for a
+	// language that wouldn't normally require one (e.g. Rust), and do evil
+	// things. So we should notify the user and confirm they would like to
+	// continue with the build.
+	if !c.Flags.AcceptCustomBuild && toolchain == "custom" {
+		customBuildMsg := "This project has a custom build script defined in the fastly.toml manifest. "
+		if c.Globals.Flag.Verbose {
+			text.Info(out, customBuildMsg)
+			customBuildMsg = ""
+		}
+		label := fmt.Sprintf("%sAre you sure you want to continue with the build step? [y/N] ", customBuildMsg)
```

Comment:
Not a blocker: for symmetry, can we check if we printed customBuildMsg above and avoid printing it again?

Reply:
Nevermind this, it's already covered.